### PR TITLE
kai is da bomb

### PR DESCRIPTION
**Pull request by kaichehung**
People will heard different noise by different key. Users' key press will effect each others noise.
Will upgrade it to make sound change as duration of key press.

### CODE REVIEW

File: public/sketch.js [hunk 1]
```diff
@@ -156,3 +163,18 @@ function usernameChanged() {
     }
   });
 }
+
+
+//Make noise as the key pressed
+function setTone(){
+  osc = new p5.Oscillator();
```
Comment (79) [inline public/sketch.js:170]: @kaichehung does `osc` here need to be a global variable?

Comment (79) [reply public/sketch.js:170]: i.e. define `var osc;` before the `function setup()` method.

**Review by 79 — COMMENTED** [review]
Code looks great! Just one question

File: public/sketch.js (hunk 1 above)
Comment (kaichehung) [reply public/sketch.js:170]: oops! Anthony you are right! it should!
It works, so I don't even find there's a bug.

https://p5js.org/reference/#/p5.Oscillator

Comment (79) [reply public/sketch.js:170]: thanks for looking! haha it shouldn't work but i'll trust you and merge this now. if we run into problems later will be an easy fix.

**Pull request merged**